src/routes/Profile.jsx

```diff
@@ -0,0 +1,58 @@
+import { useSelector } from 'react-redux';
+import ListGroup from 'react-bootstrap/ListGroup';
+import styles from '../styles/Profile.module.css';
+
+const Profile = () => {
+  const {
+    missions,
+    isLoading: missionsLoading,
+    error: missionsError,
+  } = useSelector((state) => state.missions);
+
+  const {
+    rockets,
+    isLoading: rocketsLoading,
+    error: rocketsError,
+  } = useSelector((state) => state.rockets);
+
+  if (missionsError || rocketsError) return <h2>Something went wrong</h2>;
+
+  return (
+    <div className={styles.wrapper}>
+      <hr />
+      <div className={styles.profile}>
+        <div className={styles.missions}>
+          <h2>My Missions</h2>
+          {missionsLoading ? (<div>Missions Loading</div>) : (
+            <ListGroup>
+              {missions.map((mission) => (mission.reserved ? (
+                <ListGroup.Item key={mission.mission_id} className={styles.listItem}>
+                  {mission.mission_name}
+                </ListGroup.Item>
+              ) : (
+                ''
+              )))}
+            </ListGroup>
+          )}
+
+        </div>
+        <div>
+          <h2>My Rockets</h2>
+          {rocketsLoading ? (<div>Rockets Loading</div>) : (
+            <ListGroup>
+              {rockets.map((rocket) => (rocket.reserved ? (
+                <ListGroup.Item key={rocket.rocket_id} className={styles.listItem}>
+                  {rocket.rocket_id}
+                </ListGroup.Item>
+              ) : (
+                ''
+              )))}
+            </ListGroup>
+          )}
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] @team great work rendering reserved rockets and joined missions, however, the design differs slightly with given design, the my rockets list width is too small kindly try enlarging it so that the design matches the design given:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Bifabig/space-travelers-hub-react-redux/assets/61408860/db1c4eac-035a-4cee-96f2-a22417b7bdc6)  | ![image](https://github.com/Bifabig/space-travelers-hub-react-redux/assets/61408860/7c02b614-7a77-45bf-bd84-ddee5f62c5a1)

Reply:
- [ ] [optional] @team you can be enhanced by adding a placeholder message when the "My Missions" or "My Rockets" lists are empty or adding the "Cancel reservation" and "Leave Mission" buttons to the lists here